tests/suites/test_suite_cipher.nist_kw.data

```diff
@@ -268,4 +276,3 @@ auth_crypt_tv:MBEDTLS_CIPHER_AES_192_KWP:"21fb6600c1d34a74adee67612672593a86cf23
 KWP AES-256 wrap CAVS 17.4 FAIL COUNT 4 CLEN 32
 depends_on:MBEDTLS_AES_C:MBEDTLS_NIST_KW_C
 auth_crypt_tv:MBEDTLS_CIPHER_AES_256_KWP:"c32cb3e1e41a4b9f4de79989957866f5dd48dba38c22a6ebb80e14c84bdd9534":"":"":"c29b05c2619a58ecc1d239e7a34273cd":"":"FAIL":"":0
-
```

Comment:
Did you remove an endline or an empty line here? Not sure how github shows that.

Reply:
There was an empty line at the end of the file and that commit removed it. I think the Github rendering is unambiguous; what's hard to distinguish is trailing whitespace. Github would show a file with a missing endline differently.